lightning/src/ln/channelmonitor.rs

```diff
@@ -1054,8 +1078,8 @@ impl<ChanSigner: ChannelKeys + Writeable> ChannelMonitor<ChanSigner> {
 impl<ChanSigner: ChannelKeys> ChannelMonitor<ChanSigner> {
 	pub(super) fn new(keys: ChanSigner, shutdown_pubkey: &PublicKey,
 			our_to_self_delay: u16, destination_script: &Script, funding_info: (OutPoint, Script),
-			their_htlc_base_key: &PublicKey, their_delayed_payment_base_key: &PublicKey,
-			their_to_self_delay: u16, funding_redeemscript: Script, channel_value_satoshis: u64,
+			remote_htlc_base_key: &PublicKey, remote_delayed_payment_base_key: &PublicKey,
+			on_remote_csv: u16, funding_redeemscript: Script, channel_value_satoshis: u64,
```

Comment:
Hmm, on_remote_csv definitely makes more sense than their_to_self_delay, but maybe call it remote_tx_to_remote_csv, or so? Characters don't cost us much :)

Reply:
`on_remote_tx_csv` ? Imo `remote_tx_to_remote_csv` doesn't make sense because even if are processing a remote tx we are doing so from the viewpoint of local, this csv matters us as _local_. 

Overall, either is fine but to avoid confusion, it's more important to be consistent on this with every variable names.

Reply:
I'm ok with that, though I prefer something that indicates that its the CSV on the output to our counterparty on a remote tx.